README-draft.md

```diff
@@ -0,0 +1,204 @@
+# Wit/Oracle Solidity Framework -- Contracts, SDK and CLI tools for EVM-compatible chains
```

Comment:
Is this draft ready to review? Should be replacing README.md?

Reply:
Sorry, not ready yet.